### PR TITLE
disable batch update dmls

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve?

Issue Number: ref #8079 

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

If you don't think this PR needs a release note then fill it with `None`.
```

### CODE REVIEW

File: cdc/sink/mysql/mysql.go [hunk 1]
```diff
@@ -925,10 +925,13 @@ func (s *mysqlSink) batchSingleTxnDmls(
 
 	// handle update
 	if len(updateRows) > 0 {
+		// use sql.GenUpdateSQL to generate update sql after we optimize the func.
```
Comment (amyangfei) [inline cdc/sink/mysql/mysql.go:928]:
```suggestion
		// TODO: use sql.GenUpdateSQL to generate update sql after we optimize the func.
```

**Review by amyangfei — COMMENTED** [review]
rest LGTM

**Review by amyangfei — APPROVED** [review]

**Review by hi-rustin — APPROVED** [review]

**Pull request merged**